pkgs/os-specific/linux/open-isns/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, openssl, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  name = "open-isns-${version}";
+  version = "0.95";
+
+  src = fetchFromGitHub {
+    owner = "gonzoleeman";
+    repo = "open-isns";
+    rev = "v${version}";
+    sha256 = "1c2x3yf9806gbjsw4xi805rfhyxk353a3whqvpccz8dwas6jajwh";
+  };
+
+  propagatedBuildInputs = [ openssl ];
+  outputs = ["out" "lib" ];
+  outputInclude = "lib";
+
+  installPhase = ''
```

Comment:
What about using `installFlags = "etcdir=$(out) vardir=$(out)/var/lib/isns` and `installTargets = "install install_hdrs install_lib"` instead of overriding the entire installPhase? Doesn't really matter, though.